recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "srwpy" %}
-{% set version = "0.1" %}
+{% set version = "0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/Synchrotron-Radiation-Workshop/SRW/archive/v{{ version }}.tar.gz
-  sha256: 53bdcb0c02c59659bd0cb3849eac15c166db65b69d6fb62d4b8bf97cc4c96be8
+  sha256: 5b4d6969deb5d3965b9dcad50ba509ef4ed0cf342b4380eb25760705b1f3dde3
```

Comment:
Can we move the sha256 value to the set block on top and use the templated string here? I normally do it for the recipes I am maintaining, so it's easier to have the change of version and hash in one chunk.

Reply:
Thanks @mrakitin for the review. Addressed via https://github.com/conda-forge/srwpy-feedstock/pull/5/commits/ea7b49bbf4903e162682a2da4f769b3491ee8f1a